### PR TITLE
Fixed wrapper in Disha’s PR

### PR DESCRIPTION
**Pull request by dellaert**
Jerred, I already made the wrapper work. Please review, approve if appropriate, and merge into Disha's branch. send me an email when done.

### CODE REVIEW

**Comment by dellaert** [comment]
@jerredchen please review?

**Review by jerredchen — COMMENTED** [review]
@dellaert Is `testSpiderWalking` able to run by removing the default argument in `contactPointObjectives`?

File: gtdynamics.i [hunk 1]
```diff
@@ -839,33 +840,41 @@ class Trajectory {
...
   std::vector<int> finalTimeSteps() const;
   size_t phaseIndex(size_t p) const;
   const Phase &phase(size_t p) const;
   size_t getStartTimeStep(size_t p) const;
   size_t getEndTimeStep(size_t p) const;
   const gtdynamics::PointOnLinks &getPhaseContactLinks(size_t p) const;
   std::vector<string> getPhaseSwingLinks(size_t p) const;
-  PointGoalFactor pointGoalFactor(const string &link_name,
+  PointGoalFactor pointGoalFactor(const gtdynamics::Robot &robot,
+                                  const string &link_name,
```
[the remark this answers is not in the document]
Comment (dellaert) [reply gtdynamics.i:864]: Not this PR

**Comment by jerredchen** [comment]
@dellaert Just did minor fix to allow the python wrapper to compile. Will merge into Disha's branch.

**Comment by dellaert** [comment]
Awesome, thanks! I think you need to approve before you can merge.

**Pull request merged**